.gitmodules

```diff
@@ -13,4 +13,4 @@
 [submodule "lib/risc0"]
 	path = lib/risc0
 	url = https://github.com/risc0/risc0
-	rev = "4652f9c"
+	branch = "release-0.19"
```

Comment:
Do we even need this anymore?

Reply:
Yeah its how the solidity code is imported.